infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogEntrySinkBufferManager.java

```diff
@@ -84,6 +84,7 @@ public void processBuffer() {
             LogReplicationEntryMetadataMsg metadata = entry.getMetadata();
             if (metadata.getTimestamp() <= lastProcessedSeq) {
                 buffer.remove(metadata.getPreviousTimestamp());
+                log.warn("Removed {} from the buffer with no processing", metadata.getPreviousTimestamp());
```

Comment:
Can we change this for? log.warn("Remove entry without processing, ts={}, lastProcessed={}", metadata.getEntryType(), lastProcessedSeq);